parts/k8s/cloud-init/nodecustomdata.yml

```diff
@@ -46,7 +46,7 @@ write_files:
 {{end}}
 
 - path: /etc/ssh/sshd_config
-  permissions: "0644"
+  permissions: "0600"
```

Comment:
This is server-wide ssh client configuration and should be readable by all. Is this file explicitly included in CIS?

Reply:
sshd_config should be only for sshd server component, no?